GUI/MainWindow.xaml.cs

```diff
@@ -427,8 +428,26 @@ private void EverythingRunnerExceptionHandler(Task obj)
                 Exception e = obj.Exception;
                 while (e.InnerException != null) e = e.InnerException;
                 var message = "Run failed, Exception: " + e.Message;
-                MessageBox.Show(message);
+                var messageBoxResult = System.Windows.MessageBox.Show(message + "\n\nWould you like to report this crash?", "Runtime Error", MessageBoxButton.YesNo);
                 outRichTextBox.AppendText(message + Environment.NewLine);
+                Exception exception = e;
+
+
+                if(messageBoxResult == MessageBoxResult.Yes)
+                {
+                    string body = exception.Message + "%0D%0A" + exception.Data +
+                       "%0D%0A" + exception.StackTrace +
```

Comment:
What's %0D%0A

Reply:
These are the characters for 'newline' needed to format with the line endings that emails use. For some reason '\n' , '\r\n', and 'Environment.newline'  don't work for email line endings.